packages/shorebird_cli/lib/src/commands/release/android_releaser.dart

```diff
@@ -63,7 +60,7 @@ class AndroidReleaser extends Releaser {
 
   @override
   Future<void> assertArgsAreValid() async {
-    argResults.file('public-key-path')?.assertExists();
+    argResults.assertPublicKey();
```

Comment:
This could maybe have a more verbose name. `assertPublicKey` suggests that this will fail if no public key is provided.

Reply:
Makes sense, I have renamed, lmk what you think